workers/unity/Assets/Gdk/Core/Error/LoggingDispatcher.cs

```diff
@@ -0,0 +1,23 @@
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class LoggingDispatcher : ILogDispatcher
+    {
+        public void HandleLog(LogType type, LogEvent logEvent)
+        {
+            switch (type)
```

Comment:
should we also handle `LogType.Assert` and `LogType.Exception`

Reply:
Added both, the only issue is that we can't use Debug.LogException because that needs an exception not a string. Will use Debug.LogError in that case.